rs/src/management/http_client.rs

```diff
@@ -458,7 +466,16 @@ impl TunnelManagementClient {
         match get_policy_header_value() {
             Ok(Some(policy_header_value)) => {
                 if let Ok(header_value) = HeaderValue::from_maybe_shared(policy_header_value) {
-                    headers.insert("User-Agent-Policies", header_value);
+                    let header_value_str = header_value
+                        .to_str()
+                        .unwrap()
+                        .replace("%22", "")
+                        .replace("%3B", ";");
```

Comment:
We already use the urlencoding module in this crate, you should be able to use it here: https://docs.rs/urlencoding/latest/urlencoding/#